keyboards/doio/kb3x/config.h

```diff
@@ -0,0 +1,29 @@
+/* Copyright 2022 DOIO
+ * Copyright 2022 DOIO2022 <https://github.com/DOIO2022>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+/* Ecoder */
+#define ENCODER_DEFAULT_POS 0X03
+
+/* RGB Matrix config */
+#define RGB_MATRIX_LED_COUNT 6
+#define RGB_MATRIX_DEFAULT_MODE RGB_MATRIX_CYCLE_UP_DOWN
+#define RGB_DISABLE_WHEN_USB_SUSPENDED
+#define RGB_MATRIX_KEYPRESSES
+#define RGB_MATRIX_FRAMEBUFFER_EFFECTS
+
```

Comment:
both are enabled when rgb_matrix animations are defined
```suggestion
```

Reply:
Pretty sure that isn't a thing. Must be confusing that for `rgblight`, which was deprecated.

Reply:
https://github.com/qmk/qmk_firmware/pull/21703

Reply:
That's for `develop`, which won't merge with `master` until the 26th, and I didn't happen to see any other PR that's meant to remove said lines. So, _for right now at least_, those two lines are needed.

Reply:
Incorrect, it was merged last cycle so is available on master right now.

Reply:
@zvecr  Hi, I've commited the suggestion to delete the two lines , that's right?  Thank you!